book/src/frequently_asked_questions.md

```diff
@@ -152,4 +175,4 @@ Don't [ask](https://www.youtube.com/watch?v=0QaAKi0NFkA). They just
 
 ## Why won't you take this FAQ thing seriously?
 
-Look, people just haven't asked many questions yet.
+Look, people just haven't asked many questions yet. Why are you still reading this?
```

Comment:
Change the sassy part to "Sorry, there are no easter eggs in this document, but there may be elsewhere 🥚"